e2e/tests/examples/data-driven.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { config } from 'resources/workbench-config';
+import GoogleLoginPage from 'app/page/google-login';
+import { withPage } from 'libs/page-manager';
+
+describe('Login tests:', () => {
+  const urls = [
+    { text: `${config.uiBaseUrl}${config.workspacesUrlPath}` },
+    { text: config.uiBaseUrl },
+    { text: `${config.uiBaseUrl}${config.profileUrlPath}` }
+  ];
+
+  test.each(urls)('Redirects to login page', async (url) => {
+    await withPage()(async (page) => {
```

Comment:
I couldn't get `test('', withPage()(async (page, browser) => { ...})` syntax to compile.